.github/workflows/ci.yml

```
@@ -35,7 +35,7 @@ jobs:
         continue-on-error: ${{ matrix.version == 'nightly' }}
       - uses: julia-actions/julia-processcoverage@v1
         continue-on-error: true
-      - uses: codecov/codecov-action@v3
+      - uses: codecov/codecov-action@v4
         continue-on-error: true
         with:
```

From the changelog:

> Tokenless uploading is unsuported

Thus in this form codecov does not work: https://github.com/JuliaMath/SpecialFunctions.jl/actions/runs/7779640632/job/21211100068?pr=464#step:8:31

```suggestion
        with:
          token: ${{ secrets.CODECOV_TOKEN }} # required
```

---

Wait, does it mean we need to obtain a token for all repositories where we want to upload coverage stats?

---

If you want to use codecov-action@v4, yes, that's my understanding based on the README and the changelog.